polkadot/node/network/collator-protocol/src/validator_side_experimental/common.rs

```diff
@@ -46,6 +54,19 @@ pub const INACTIVITY_DECAY: u16 = 1;
 /// Maximum number of stored peer scores for a paraid. Should be greater than
 /// `CONNECTED_PEERS_PARA_LIMIT`.
 pub const MAX_STORED_SCORES_PER_PARA: u8 = 150;
+
+/// Slashing value for a failed fetch that we can be fairly sure does not happen by accident.
+pub const FAILED_FETCH_SLASH: Score = Score::new(20).expect("20 is less than MAX_SCORE");
+
+/// Slashing value for an invalid collation.
+pub const INVALID_COLLATION_SLASH: Score = Score::new(1000).expect("1000 is less than MAX_SCORE");
+
+/// Minimum reputation threshold that warrants an instant fetch.
```

Comment:
What does instant fetch mean ? What if multiple collators have same rep ?

Reply:
Fetches can be delayed if the peer's reputation is below this threshold (and there are no other collations from peers with higher reps). This is so that peers with higher reputations could connect and advertise their collations in the meantime. Otherwise the reputation system would only be useful for connection triaging.

> What if multiple collators have same rep ?

Just pick one. This shouldn't normally happen, as Aura ensures unique authors